pages/pil/update/views/training-data.jsx

```diff
@@ -42,11 +42,7 @@ class TrainingData extends Component {
               <div className="govuk-grid-column-one-quarter">
                 <ul className="actions">
                   <li>
-                    <form method="POST" noValidate>
-                      <input type="hidden" name="trainingModuleId" value={certificate.id} />
-                      <input type="hidden" name="action" value="delete" />
-                      <button type="submit" className="link"><span><Snippet>actions.remove</Snippet></span></button>
-                    </form>
+                    <a href={`${url}/training/${certificate.id}/delete?referrer=${url}`}><Snippet>actions.remove</Snippet></a>
```

Comment:
I don't _like_ performing data manipulation on a `GET` - especially deleting data.

Reply:
yeah, I know what you mean.